### PR TITLE
S4t5 session logic

### PR DESCRIPTION
**Pull request by mikeycodingstuff**
added dunk and flunk functionality to details page and added logic to increment/decrement in db. changed icons and styling

### CODE REVIEW

File: Test/ViewHelpers/BiscuitViewHelperTest.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ public function testSuccessDisplayAllBiscuits()
...
+        $exp = '<div id="3" class="card d-flex flex-direction-column align-items-center p-3 m-4 col-10 col-lg-3"><div class="card-title card-background rounded"><h2 class="text-center p-2">Digestive</h2></div><div class="card-img d-flex justify-content-center mb-3"><img src="img.jpg" class="rounded mw-100" alt="Digestive" /></div><form action="biscuitdetails.php" method="GET"><input type="hidden" name="id" value="3" /><button type="submit" class="btn btn-light">More Info</button></form><div class="d-flex container-fluid justify-content-around"><form action="hiddenDunk.php" method="POST"><input type="hidden" name="id" value="3" ><button type="submit"class="btn btn-success"><img class="list-icon" src="design/Dunk_Icon.png" /></button></form><form action="hiddenFlunk.php" method="POST"><input type="hidden" name="id" value="3" ><button type="submit"class="btn btn-danger"><img class="list-icon" src="design/Flunk_Icon.png" /></button></form></div></div>';
         $actual = BiscuitViewHelper::displayAllBiscuits($input, $input2);
```
Comment (aidanmaskell) [inline Test/ViewHelpers/BiscuitViewHelperTest.php:20]: maybe try to stay consistent with line breaks in these test functions, this one has no line breaks but the bottom few have between mocks, expected and actual variables

File: hiddenDunk.php [hunk 2]
```diff
@@ -10,7 +10,15 @@
...
+        $_SESSION['dunkFlunk'][$id] = true;
+        BiscuitHydrator::decrementFlunk($db, $id);
+        BiscuitHydrator::incrementDunk($db, $id);
+        header('Location: index.php#' . $id);
+    } else {
+        $_SESSION['dunkFlunk'][$id] = true;
+        BiscuitHydrator::incrementDunk($db, $id);
+        header('Location: index.php#' . $id);
+    }
+
```
Comment (aidanmaskell) [inline hiddenDunk.php]: line break could be removed

**Review by aidanmaskell — APPROVED** [review]
all looks really good, just a couple of line breaks here or there but other than that its great

File: index.php [hunk 3]
```diff
@@ -26,7 +26,7 @@
 </head>
```
Comment (aidanmaskell) [inline index.php]: could be removed

File: Test/ViewHelpers/BiscuitViewHelperTest.php (hunk 1 above)
Comment (mfk07) [reply Test/ViewHelpers/BiscuitViewHelperTest.php:20]: extra breaks removed

File: hiddenDunk.php (hunk 2 above)
Comment (mfk07) [reply hiddenDunk.php]: breaks removed

File: index.php (hunk 3 above)
Comment (mfk07) [reply index.php]: break removed

**Review by aidanmaskell — APPROVED** [review]

**Pull request merged**